src/components/Sortable/list/index.tsx

```diff
@@ -0,0 +1,131 @@
+
+import { SortableItem } from './Item';
+
+import { Item, RenderProps } from './types';
+
+type Props = {
+    items: any[];
+    // eslint-disable-next-line no-unused-vars
+    onChange: (items: any[]) => void;
+    // eslint-disable-next-line no-unused-vars
+    renderItem: ({ item, index, handleProps }: RenderProps) => ReactNode;
+    handle?: boolean;
+    disabled?: boolean;
+};
+
+const useStyles = makeStyles(theme => ({
+    list: {
+        padding: 0,
+        margin: 0,
+        listStyleType: 'none',
+    },
+    draggableItem: {
+        padding: theme.spacing(1),
+        backgroundColor: theme.palette.grey['200'],
+        // @ts-ignore
+        border: `1px solid ${theme.palette.grey['400']}`,
+        borderRadius: 5,
+        boxShadow: '-2px 8px 3px -3px rgba(0,0,0,0.15)',
+    },
+}));
+
+export const SortableList: FunctionComponent<Props> = props => {
+    const { items, onChange, handle = false, disabled, renderItem } = props;
+    const [activeItem, setActiveItem] = useState<Active | undefined>();
+    const classes = useStyles();
+    const sensors = useSensors(
+        useSensor(PointerSensor),
+        useSensor(KeyboardSensor, {
+            coordinateGetter: sortableKeyboardCoordinates,
+        }),
+    );
+
+    const handleDragEnd = useCallback(
+        (event: DragEndEvent) => {
+            setActiveItem(undefined);
+            const { active, over } = event;
+            if (over && active.id !== over?.id) {
+                const oldIndex: number = items.findIndex(
+                    (item: Item) => item.id === active.id,
+                );
+                const newIndex: number = items.findIndex(
+                    (item: Item) => item.id === over.id,
+                );
+                const newItems = arrayMove(items, oldIndex, newIndex);
+                onChange(newItems);
+            }
+        },
+        [items, onChange],
+    );
+
+    return (
+        <DndContext
+            sensors={sensors}
+            collisionDetection={closestCenter}
+            onDragEnd={handleDragEnd}
+            onDragStart={({ active }) => {
+                setActiveItem(active);
+            }}
+            modifiers={[restrictToVerticalAxis]}
+        >
+            <SortableContext
+                disabled={disabled}
+                items={items}
+                strategy={verticalListSortingStrategy}
+            >
+                <ul className={classes.list}>
+                    {items.map((item, index) => (
+                        <SortableItem
+                            handle={handle}
+                            key={item.id}
+                            id={item.id}
+                        >
+                            {handleProps =>
+                                renderItem({
+                                    item,
+                                    index,
+                                    handleProps,
+                                })
+                            }
```

Comment:
isn't that the same as something like:

```javascript
handleProps => <RenderItem item={item} index={index} handleProps={handleProps}/>
```

?

If so, I prefer if we use JSX notation for components, I find it more legible. Nothing blocking
If not, my bad